test/test_cuda.py

```diff
@@ -2674,15 +2674,6 @@ def test_diagflat(self):
     def test_norm(self):
         _TestTorchMixin._test_norm(self, device='cuda')
 
-    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
-    @unittest.skipIf(not TEST_MAGMA, "no MAGMA library detected")
-    def test_nuclear_norm_axes_small_brute_force(self):
-        _TestTorchMixin._test_nuclear_norm_axes(self, device='cuda')
-
-    @unittest.skipIf(not TEST_MAGMA, "no MAGMA library detected")
-    def test_nuclear_norm_exceptions(self):
```

Comment:
Could you try adding the `@skipCUDANonDefaultStreamIf(True)` decorator here and check whether the tests pass? I think this might be an issue.

Reply:
I'm just guessing here based on your comments about the tests passing after being moved to a different script.